src/v0/destinations/clevertap/transform.js

```diff
@@ -291,7 +291,7 @@ const responseBuilderSimple = (message, category, destination) => {
     }
     eventPayload.type = 'event';
     // stringify the evtData if it's an Object or array.
-    if (eventPayload.evtData) {
+    if (eventPayload.evtData && eventPayload.evtName !== 'Charged') {
```

Comment:
We have to skip `Items` property only from stringification. All other properties should be a string. And we should also check if `Items` property value is an array of objects.

Reply:
Added check